src/dos/dos_keyboard_layout.cpp

```diff
@@ -195,7 +192,9 @@ static uint32_t read_kcl_file(const FILE_unique_ptr &kcl_file, const char *layou
 
 	const auto seek_pos = 7 + rbuf[6];
 	if (fseek(kcl_file.get(), seek_pos, SEEK_SET) != 0) {
-		LOG_WARNING("LAYOUT: could not seek to byte %d in keyboard layout file: %s", seek_pos, strerror(errno));
+		LOG_WARNING("DOS: Could not seek to byte %d in keyboard layout file: %s",
```

Comment:
That's a very specific error message. Does the use need to know the exact location of the byte? 😅 Does he care? 😆 Suggest changing it to something like `DOS: Error reading keyboard layout file: '%'`